.devcontainer/devcontainer.json

```
@@ -3,7 +3,7 @@
 {
 	"name": "Go",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/go:1-1.21-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/go:1-1.22-bullseye",
```

```suggestion
	"image": "mcr.microsoft.com/devcontainers/go:1-1.22-bookworm",
```

Can you test if dev container is created fine and steps in README work?

---

tested the new image tag with all the `things to try`. it is working fine.
shall I change this to bookworm then?

---

> shall I change this to bookworm then?

Yep!